package/yast2-firstboot.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Jul  3 10:58:27 UTC 2019 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Remove the references to the already dropped automatic
+  configuration feature (FATE#314695).
```

Comment:
code wise good, I am just not sure if QAM will be happy about so late feature in SLE15 GA.

Reply:
Well, basically we are just dropping unused (and confusing) stuff from the example. Perhaps a maintenance update does not make sense form them. Let's see.